docs/common/istio-ambient-mode.md

```diff
@@ -0,0 +1,354 @@
+## Istio Ambient mode
+
+This documentation is specific to the Istio Ambient mode using Sail Opeartor installation. These features provide early access to upcoming product features, enabling users to test functionality and provide feedback during the development process.
+
+## Introduction to Istio Ambient mode
+
+Ambient mesh and its reference implementation with Istio's ambient mode was announced in September 2022. It reaches a stable status in Istio 1.24 release. The core innovation behind ambient mesh is that it slices Layer 4(L4) and Layer 7(L7) processing into two distinct layers. Istio's ambient mode is powered by lightweight, shared L4 node proxies and optional L7 proxies, removing the needfor traditional sidecar proxies from the data plane.
+
+The lightweight shared L4 node proxy is called the `ztunnel`(zero-trust tunnel). ztunnel drastically reduces the overhead of running a mesh by removing the need to potentially over-provision memory and CPU within a cluster to handle expected loads, while still providing security using mutual TLS with cryptographic identity, simple L4 authorization policies, and telemetry.
+
+The L7 proxies are called `waypoints`. Waypoints process L7 functions such as traffic routing, authorization policy enforcement, and resilience. Waypoints run outside of your application deployments and can scale independently based on your needs.
+
+In contrast to the earlier injection of sidecars. Users can start with the secure L4 overlay, which offers features such as mTLS, authorization policy, and telemetry. Complex L7 handling such as retries, traffic splitting, load balancing, and observability collection can then be enabled on a case-by-case basis.
+
+## Component version
+
+For running Istio ambient mode on OpenShift, we recommend using Sail Operator and installing `Istio`, `IstioCNI` and `ZTunnel` resources version v1.24.0 or newer.
```

Comment:
As this documentation is for Sail Operator, we can be generic. What's your though about the following sentence.

```suggestion
To install Istio Ambient mode using Sail Operator, use version `v1.24.0` or later and deploy the `Istio`, `IstioCNI`, and `ZTunnel` resources.
```

Reply:
I think the version `v1.24.0` or later is only required when installing it on OpenShift. I will rephrase it as :

```
To install Istio Ambient mode using Sail Operator on OpenShift, use version `v1.24.0` or later and deploy the `Istio`, `IstioCNI`, and `ZTunnel` resources.
```